### DISCUSSION TITLE
umi4 使用@umijs/preset-vue 时，支持mpa模式吗？

### DISCUSSION THREAD

**Discussion by wuhaoxiangfau** [umijs/umi#11437] [Q&A] [answered]
umi4 使用@umijs/preset-vue 时，支持mpa模式吗？
在.umirc.ts 是可以通过mpa参数继续配置，还是通过chainWebpack 去更改entry

**Comment by fz6m** [comment]
不支持，目前 mpa 和 vue 都是独立的功能，无法和其他功能一起使用。

**Reply by fz6m** [reply, marked as answer]
可以自己实现，思路参考 https://github.com/umijs/umi/issues/11438#issuecomment-1641869349
